### PR TITLE
Dm 3528

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3528

## Description - what does this code do?
Adds four new tables to the schema (others to follow in part 2):
- `Communities`
- `CommunityLeaders`
- `CommunityFaqs`
- `CommunityPractices`

## Testing done - how did you test it/steps on how can another person can test it 
1. Run rails db:migrate and make sure your local DB is updated properly.

## Acceptance criteria
- Review Community Diagram
- Build Community Feature Branch

## Definition of done
- [X] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: spec/requests/communities_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "Communities", type: :request do
```
Comment (ahay-agile6) [inline spec/requests/communities_spec.rb:3]: do we need this file as well?

**Review by ahay-agile6 — APPROVED** [review]
LGTM. just a couple of questions

File: app/assets/stylesheets/communities.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Communities controller here.
+// They will automatically be included in application.css.
```
[the remark this answers is not in the document]
Comment (JoshingYou1) [reply app/assets/stylesheets/communities.scss:2]: Nope. Removed! 😄

File: spec/requests/communities_spec.rb (hunk 1 above)
Comment (JoshingYou1) [reply spec/requests/communities_spec.rb:3]: Nope. Removed! 😄

**Review by bradjohnson92008 — APPROVED** [review]
LGTM - thx Josh

**Review by camillevilla — COMMENTED** [review]
✅ This looks like a solid start to me! 
❓ Do we still need the text fields for `question` and `answer` in community FAQs? It sounded like those can be handled by PB accordion components. 
Screen Shot 2022-08-18 at 2 53 45 PM

**Comment by bradjohnson92008** [comment]
This is obsolete

**Pull request closed without merging**